vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopWallet.kt

```diff
@@ -177,12 +177,12 @@ class OidcSiopWallet(
      * or JSON serialized as a JWT Request Object.
      */
     suspend fun createAuthnResponse(
-        request: AuthenticationRequestParametersFrom<*>
+        request: AuthenticationRequestParametersFrom
     ): KmmResult<AuthenticationResponseResult> = catching {
         val response = createAuthnResponseParams(request).getOrThrow()
         if (request.parameters.responseType == null
-            || (!request.parameters.responseType.contains(ID_TOKEN)
-                    && !request.parameters.responseType.contains(VP_TOKEN))
+            || (!request.parameters.responseType!!.contains(ID_TOKEN)
+                    && !request.parameters.responseType!!.contains(VP_TOKEN))
```

Comment:
please refactor to avoid !!
Possible workaround: val responseType = request.parameters.responseType; if(responseType == null ...)